Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1

```diff
@@ -57,9 +57,15 @@ function Initialize-AzureSubscription {
             $additional['CurrentStorageAccountName'] = $StorageAccount
         }
 
+        $EnvironmentName = "AzureCloud"
```

Comment:
Can you case this variable `$environmentName`? That will make it consistent with the rest of the file and follow the convention that uppercase means a variable was passed-in via params to the function.